lizard_auth_server/views_api_v2.py

```diff
@@ -452,7 +451,7 @@ def form_valid(self, form):
             An error 400 when mandatory keys are missing from the decoded
             JWT message or when the language is unknown.
 
-            An error 500 when a duplicate username is found.
+            An error 409 when a duplicate username is found.
```

Comment:
Even tussen haakjes bijzetten wat de officiële text van een 409 is: ("conflict"). Zoiets.

Reply:
Goed punt, done!